cmd/validate_image.go

```diff
@@ -101,15 +104,28 @@ Use an EnterpriseContractPolicy resource from the currently active kubernetes co
 
 Use an EnterpriseContractPolicy resource from a different namespace:
 
-  ec validate image --image registry/name:tag --policy my-namespace/my-policy`,
-		PreRunE: func(cmd *cobra.Command, args []string) error {
-			s, err := applicationsnapshot.DetermineInputSpec(data.filePath, data.input, data.imageRef)
-			if err != nil {
-				return err
+  ec validate image --image registry/name:tag --policy my-namespace/my-policy
+
+Use an inline EnterpriseContractPolicy spec
+  ec validate image --image registry/name:tag --policy '{"publicKey": "<path/to/public/key>"}'`,
+		PreRunE: func(cmd *cobra.Command, args []string) (allErrors error) {
```

Comment:
I _think_ I'm wrong on this, but just to check if `applicationsnapshot.DetermineInputSpec()` fails and `policy.NewPolicy` fails, do we still want to proceed?
Also if one of these fails but the other succeeds are we okay with returning the error?

Reply:
The behavior I was aiming for was: if any, or both, checks fail then it's a fatal error. Added a unit test to confirm this behavior.